### PR TITLE
Adding support for HEVC video codec

### PR DESCRIPTION
**Pull request by KarlGallagher**
Using same FFmpeg parameters as h264 for the most part but letting the encoder choose profile and level automatically.
Bitrate settings mirror vp9

tested on macOS 10.15.7 and Ubuntu 20.04 LTS (FFmpeg 4.4)

### CODE REVIEW

**Comment by KarlGallagher** [comment]
@googlebot I signed it!

**Review by joeyparrish — COMMENTED** [review]
Looks good to me.  Thank you for the contribution!  As soon as you have addressed the CLA issue, I think it should be okay to merge this.  Please let me know if you need any assistance on the CLA.

File: streamer/bitrate_configuration.py [hunk 1]
```diff
@@ -114,7 +120,7 @@ def get_output_format(self) -> str:
     # TODO(#31): add support for configurable output format per-codec
     if self.get_base_codec() == VideoCodec.VP9:
       return 'webm'
-    elif self.get_base_codec() == VideoCodec.H264:
+    elif self.get_base_codec() == VideoCodec.H264 or VideoCodec.HEVC:
```
Comment (TheModMaker) [inline streamer/bitrate_configuration.py:123]: This doesn't do what you want.  You need to do `self.get_base_codec() == VideoCodec.HEVC`; or you can replace it with `self.get_base_codec() in {VideoCodec.H264, VideoCodec.HEVC}`.

**Comment by KarlGallagher** [comment]
@joeyparrish I think ill have to close and reopen a new PR to resolve issues with my primary email.

@TheModMaker  - thanks, will incorporate the suggested changes in the new PR

**Pull request closed without merging**